std/getopt.d

```diff
@@ -413,7 +413,7 @@ getopt(args,
 An unrecognized option such as "--baz" will be found untouched in
 $(D args) after $(D getopt) returns.
 
-$(D Help Information Generation)
+$(B Help Information Generation)
```

Comment:
I'm going with that. I have absolutely zero idea how to build the phobos ddocs, so I can't give you a preview. There seems to be many cross dependencies between phobos and dlang.org, and I don't know how to set that up. I really REALLY think there should be a way to say "make html" and it works if you have some dependencies cloned from dlang.org there.